Added socios de trabajo images

ValeriaAcuM opened the pull request:
En la carpeta de imágenes venían algunas imágenes que se repiten en Our Institutions y no las agregué en esta sección de socios de trabajo, ya que se presentan en la sección de arriba.
El logo del MAUCC no se ve porque tiene letras blancas.

Así se ve en mi local:
![image](https://github.com/cetav-ddw/lalibertad-cetav/assets/106450335/12840c2c-737d-4140-80ac-b8b5dbec8d55)

ValeriaAcuM committed:
Added socios de trabajo images

jleiva commented:
@ValeriaAcuM hay que jugar con las proporciones, porque algunos logos se ven grandes y otros no se pueden ni leer

<img width="1426" alt="image" src="https://github.com/cetav-ddw/lalibertad-cetav/assets/598867/49d1cbf7-6e7b-4f6a-b654-495b81721481">

Igna24 committed:
Merge branch 'main' into sociosTrabajo-Images

ValeriaAcuM committed:
Update images sizes OurAllies.jsx

DefArs committed:
Merge branch 'main' into sociosTrabajo-Images

jleiva commented:
@ValeriaAcuM proba si con este componente se puede tener un mejor ratio, https://chakra-ui.com/docs/components/aspect-ratio/usage porque se siguen viendo desproporcionadas.

jleiva commented on src/app/components/OurAllies.jsx:
```diff
@@ -13,12 +13,16 @@ const OurAllies = ({ images }) => {
           >
+            <Image
+              src={imageObj.url}
+              alt={imageObj.description}
```
o usamos el alt, o el description en linea 26, no se pueden tener ambos

ValeriaAcuM committed:
Update OurAllies.jsx

@jleiva no lo logre con ratio, ya que cortaba las imagenes y no pude ajustarlas al contenedor. Lo que hice fue ajustar los valores del height y widht del box

Igna24 committed:
Merge branch 'main' into sociosTrabajo-Images

ValeriaAcuM committed:
Update sizes OurAllies.jsx and OurInstitutions.jsx

ValeriaAcuM committed:
update OurAllies.jsx

ValeriaAcuM committed:
Update OurAllies.jsx

DefArs committed:
Merge branch 'main' into sociosTrabajo-Images

Igna24 committed:
Merge branch 'main' into sociosTrabajo-Images

ValeriaAcuM committed:
Update order imageData.js, institutionsData.js
